package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@babel/standalone": "^7.23.3",
-    "@clerk/nextjs": "^4.27.1",
```

Comment:
I don't think you need to change the versions too
But no worries, I'll ignore them during merge.